mindmeld/components/nlp.py

```diff
@@ -781,8 +785,8 @@ def __init__(self, app_path, domain, resource_loader=None, progress_bar=None):
 
         # If there is only one intent in the domain, the classifier would not run
         # hence we only account for classifiers were there are two or more intents
-        if len(intents) > 1:
-            self.progress_bar = progress_bar
+        self.progress_bar = progress_bar
+        if len(intents) > 1 and self.progress_bar is not None:
```

Comment:
```suggestion
        if len(intents) > 1 and self.progress_bar:
```

Reply:
I cannot do this is `tqdm` objects pass that check but are not None objects ie when self.progress_bar exists,  `if self.progress_bar` returns False but `if self.progress_bar is not None` return True.